Added a document with background/howtos on condition mapping table

thetaurean opened the pull request:
This PR adds a markdown file with background & procedures on the condition mapping table:
- How to fetch the table to a local csv
- *TBD* How to update the table via CLI using public sources
  - pending outcome of #11510
- How to update the table via local csv
- How to create a local csv from public sources

It also includes a listing of public sources used.

Test Steps:
1. N/A

## Changes
- Added `observation-mapping-table.md` to `docs/getting-started/standard-operating-procedures`

## Checklist

### Testing
N/A

## Linked Issues
- Fixes #12149 

## Todo
- Instructions for using #11510 (pending) to update the table

mkalish commented on prime-router/docs/getting-started/standard-operating-procedures/observation-mapping-table.md:
```diff
@@ -0,0 +1,77 @@
+
+## Introduction
+
+The observation mapping table maps a plethora of conditions, observations, and aoe question to codes in various code
```
```suggestion
The observation mapping table maps a plethora of conditions, observations, and aoe questions to codes in various code
```

mkalish commented on prime-router/docs/getting-started/standard-operating-procedures/observation-mapping-table.md:
```diff
@@ -0,0 +1,77 @@
+periodically by dumping the table from staging/prod or using the CLI to update the local table.
+
+## Update prod/staging tables using RESTful sources
+TBD
```
Is there a ticket for this we can reference here?

mkalish commented on prime-router/docs/getting-started/standard-operating-procedures/observation-mapping-table.md:
```diff
@@ -0,0 +1,77 @@
+
+### Columns
+Refer to the latest [observation-mapping.csv](/prime-router/metadata/tables/local/observation-mappings.csv)
+use the following columns at your peril:
```
😄 Can we elaborate on what peril means here?  Is this just saying be careful creating a CSV be hand

thetaurean replied:
Main peril I was consider was using the columns listed in this markdown doc and failing to check the CSV.
(i.e. CSV gets updated, has new columns, markdown update is forgotten).

mkalish commented on prime-router/docs/getting-started/standard-operating-procedures/observation-mapping-table.md:
````diff
@@ -0,0 +1,77 @@
+## Create or update prod/staging tables using a local CSV
+Use the prime CLI `lookuptables create` command:
+```zsh
+./prime lookuptables create prod -i ./observation-mapping.csv -a -n observation-mapping
````
Just want to confirm that if this table already existed this is an update and doesn't drop what's there?

thetaurean replied:
Correct 👍 `create` handles that use case i.e. there is no separate `update` command.

```
Usage: prime lookuptables [OPTIONS] COMMAND [ARGS]...

  Manage lookup tables

Options:
  -h, --help  Show this message and exit

Commands:
  list             List the lookup tables
  get              Fetch the contents of a lookup table
  create           Create a new version of a lookup table
  activate         Activate a specific version of a lookup table
  diff             Generate a difference between two versions of a lookup
                   table
  loadall          Load all the tables stored as CSV in the specified
                   directory
  compare-mapping  Compares a sender compendium against an observation mapping
                   lookup table, outputting an annotated CSV
```

```
Usage: prime lookuptables create [OPTIONS]

  Create a new version of a lookup table

Options:
  -e, --env <name>       Connect to <name> environment. Choose between
                         [local|test|staging|prod]
  -i, --input-file PATH  Input CSV file with the table data
  -s, --silent           Do not generate diff or ask for confirmation
  -a, --activate         Activate the table upon creation
  -n, --name TEXT        The name of the table to perform the operation on
  --show-table           Always show the table to be created
  -f, --force            Force the creation of new table(s) even if it is
                         already exist
  -h, --help             Show this message and exit
```

mkalish reviewed (approved):
LGTM, a few questions I'd like to work through though before merging.

thetaurean commented:
I believe all feedback is addressed. Will merge this evening barring any further feedback.

arnejduranovic commented on prime-router/docs/getting-started/standard-operating-procedures/observation-mapping-table.md:
```diff
@@ -0,0 +1,77 @@
+to a row in the preceding table using its Member OID. 
+
+#### AOE Questions
+Either copy the values from the website and map them into the appropriate columns  
```
Do we need more elaboration in this section? It's not clear exactly how to do this, or is it self-explanatory?

thetaurean replied:
They are sufficiently self-explanatory I believe.

thetaurean commented on prime-router/docs/getting-started/standard-operating-procedures/observation-mapping-table.md:
```diff
@@ -0,0 +1,77 @@
+- Created At
+
+The `Value Source` column needs to be manually entered based on data source below. Current possible values:
+- LOINC.org
```
`LOINC.org` in the latest xlsx ✅ Good call on double-checking this 👍

thetaurean commented on prime-router/docs/getting-started/standard-operating-procedures/observation-mapping-table.md:
````diff
@@ -0,0 +1,77 @@
+## Create or update prod/staging tables using a local CSV
+Use the prime CLI `lookuptables create` command:
+```zsh
+./prime lookuptables create prod -i ./observation-mapping.csv -a -n observation-mapping
````
It was my understanding that the AOE questions will need to be manually updated. Not that they change frequently.

I've added a note to the effect of all of the above 👍 .

thetaurean commented:
@arnejduranovic I addressed your feedback. Also made it clearer on the TBD section that it's waiting for results of #11510.

I'll wait for your 👍 to merge.
